File: app/src/main/scala/lesson8/android/weather/Weather.scala
package lesson8.android.weather

import java.util.Date

import android.util.Xml

object Weather {
  def getWeather(xml: Xml): Weather = ???
  def getForecast(xml: Xml): Array[Weather] = ???
}

class Weather(val city: String,
              val country: String,
              val temp: String,
              val tempUnit: String,
              val cloudness: String,
              val humidity: String,
              val pressure: String,
              val wind: String,
              val date: Date) {
}

File: app/src/main/java/ru/ifmo/md/lesson8/Weather.java
package ru.ifmo.md.lesson8;

import java.util.ArrayList;

public class Weather {
    public String city = "";
    public String tempr = "";
    public String date = "";
    public String type = "";
    public String wind = "";
    public String humidity = "";
    public ArrayList<Weather5Days> weather5Days = new ArrayList<Weather5Days>();

    public Weather() {

    }

    public Weather(String city, String tempr, String date, String type, String wind, String humidity, ArrayList<Weather5Days> weather5Days) {
        this.city = city;
        this.tempr = tempr;
        this.date = date;
        this.type = type;
        this.wind = wind;
        this.humidity = humidity;
        this.weather5Days = weather5Days;
    }
}
